src/website/src/pages/ec/components/button/docs/usage.mdx

```diff
@@ -3,39 +3,74 @@ title: Usage
+## Anatomy
 
-| Button name           | Visual style                                                                                            | Usage                                                                                         |
-| --------------------- | ------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------- |
-| Call-to-action button | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_CTA_Default.png)       | for the most important action on the page like "Apply for funding" or "Watch live streaming". |
-| Primary button        | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Primary_Default.png)   | for primary actions in forms, to initiate a search query or to apply a filter.                |
-| Secondary button      | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Secondary_Default.png) | or secondary actions.                                                                         |
-| Ghost button          | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Ghost_Default.png)     | when an action does not require dominance on a page, for instance to "cancel" a form.         |
-| Search button         | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Search_Default.png)    | use exclusively for the search function.                                                      |
+| Name  | Mandatory |
+| ----- | :-------: |
+| Label |     X     |
+| Icon  |           |
 
-Each page can have one to two Primary (or Call-to-action) buttons. Any remaining calls-to-action should be represented as Secondary or Ghost buttons.
+Buttons are defined by 3 major attributes:
 
 ### Button labels
 
-Button labels tell users what will happen when they click the button. Use verbs that describe the action, such as Send, Add or Delete. Do not use capitalization on buttons (only the first word in a phrase and proper nouns are capitalized) and no more than three words for button labels.
+Button labels tell users what will happen when they click the button.
+
+- Should be clear and distinct.
+- Use verbs that describe the action, such as Send, Add or Delete.
+- Keep it short, no more than 3 words.
 
 ### Button icons
 
-It is possible to add an icon to a button to enhance the action.
+Choose an icon that reinforce or clarify the meaning of a label in that particular context.
+Icons can be on right or left of your label.
+
+### Colour
+
+Ensure that the button background colour or label colour in the case of the ghost button provides sufficient contrast with the context/section background.
+
+## When to use
+
+To trigger an action. _ Actions which affect the website's state are buttons. _
+Actions include: Search, Submit... etc.
```

Comment:
```suggestion
For example, Search, Submit, Download... etc.
```

Reply:
Download is a link, not a button.

Reply:
How to create download button https://www.w3schools.com/howto/howto_css_download_button.asp

Reply:
fun fact, the provided demo isn't working (isn't downloading) because button aren't made to download files.
We should use the download attribute made specifically to download files.
It only exists for links https://www.w3schools.com/tags/att_a_download.asp (notice this is the same source than your example), https://www.w3.org/TR/2014/REC-html5-20141028/links.html#downloading-resources

Reply:
https://styleguide.mailchimp.com/web-elements/ 
Buttons should always contain actions. The language should be clear and concise. Capitalize every word, including articles. It’s OK to use an ampersand in button copy.
so download isn't an action? If we want to compromise, we can use "search, submit, subscribe, login" instead.
Below is another reasoning...but I am willing to let go...
"A download button can make your website look significantly more professional than simply offering your download through links"

Reply:
99% of the time the "login" will simply navigate to another page containing a form. 
While it's ok to use buttons in forms it's not to use button in replacement of the link that leads to form page.
For instance on https://github.com "sing in" is a link, then it leads to a page with a sign in form which in turn contain a "sign in" button.
It's not about letting go it's about using things for what they really are and not being deceptive to our users.
https://adamsilver.io/articles/but-sometimes-buttons-look-like-links/
Again in the case of downloading files, the W3C standardised the "download" attribute which is the proper, accessible way to download files and it exists only for links.

Reply:
<button type="submit"> "submit" is also a button. Including login, register,..I was reading the same article you just listed above.
<type="button">  are used to create features...
Call to action, just regular links styled as buttons. It is visually weakened if we just use it as a link.
Links allow users to navigate to pages or locations within a page. We can open a link in a new tab, copy the address or bookmark it. Buttons don't work this way.
Here "cancel""add single comment""start a review" are all buttons. It is not easy to write about buttons.

Reply:
> We can open a link in a new tab, copy the address or bookmark it. Buttons don't work this way.

that's a very good reason to NOT style links as buttons, isn't it?

Reply:
Yes sir. The task is to enforce it. Only call to action is visually styled as a button. What about the primary button? What to use it?

Reply:
> Differentiating calls to action
> If you’ve been nodding along so far, then you’ll probably agree that calls to action need to stand out. But that they also need to look different to buttons to signify their true behaviour of a link.

— https://medium.com/simple-human/but-sometimes-links-look-like-buttons-and-buttons-look-like-links-9b371c57b3d2

CTA are mostly links. We could have a variant of the "link" component called "CTA", which would look a bit like a button, while being visually different (bigger, bolder, different colours, I don't know).

Now, there could be a case where we want a CTA button, e.g. a "subscribe to the newsletter" button in a form (if "subscribe to the newsletter" is the main action and leads to a conversion). Then maybe we will need to keep a CTA variant of the button. I'm not 100% sure. Wouldn't the primary button be a better candidate here?

Reply:
Can we actually table this topic for the time being due to our priorities and todos are still outstanding? We will improve ourselves along the way or you see this as a blocker?